src/main/java/team/asd/service/CollectionService.java

```diff
@@ -0,0 +1,54 @@
+package team.asd.service;
+
+import org.apache.commons.collections4.CollectionUtils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class CollectionService implements IsCollectionService {
+	@Override
+	public List<Object> unmodifiableList(Object... objects) {
+		if (objects.length == 0) {
```

Comment:
This condition is redundant because if objects.length == 0 then the loop at line 19 is skipped